common/c_cvarlist.cpp

```diff
@@ -332,6 +332,9 @@ CVAR(               cl_waddownloaddir, "", "Set custom WAD download directory",
 CVAR(				developer, "0", "Debugging mode",
 					CVARTYPE_BOOL, CVAR_NULL)
 
+CVAR(deb_disconnect, "0", "Show source file:line where a disconnect happens",
```

Comment:
deb_?

Reply:
Supposed to stand for `debug` but honestly I had my doubts about that.  Was trying to keep it between 1 and 3 letters.  `d_` i thought might get confused with "Doom" or something else.

Reply:
I went ahead and changed it to `debug_` based on conversations elsewhere.